### PR TITLE
blisp: add bdd as maintainer

### PR DESCRIPTION
**Pull request by bdd**
## Description of changes

Add myself as a maintainer.

Looks like current sole maintainer @fortuneteller2k is no longer on GitHub.

### CODE REVIEW

**Comment by AndersonTorres** [comment]
If this is the case, you can remove him from maintainers list.

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -2122,6 +2122,7 @@
     email = "bdd@mindcast.org";
     github = "bdd";
     githubId = 11135;
+    matrix = "@bdd:mozilla.org";
```
Comment (AndersonTorres) [inline maintainers/maintainer-list.nix:2125]: Put this in a separated commit

**Review by AndersonTorres — CHANGES_REQUESTED** [review]

File: maintainers/maintainer-list.nix (hunk 1 above)
Comment (bdd) [reply maintainers/maintainer-list.nix]: Oof. This wasn't supposed to be part of this commit.
Amended.

**Review by katexochen — APPROVED** [review]

**Review by AndersonTorres — APPROVED** [review]

**Comment by AndersonTorres** [comment]
No need for waiting ofBorg.

**Pull request merged**